openfe/tests/protocols/test_openmm_equil_rfe_protocols.py

```diff
@@ -712,7 +712,7 @@ def tyk2_xml(tmp_path_factory):
 
     tmp = tmp_path_factory.mktemp('xml_reg')
 
-    dryrun = pu.run(dry=True, basepath=tmp)
+    dryrun = pu.run(dry=True, shared_basepath=tmp)
```

Comment:
Need to also update the CLI?

Reply:
nah, cli uses `execute()`/`_execute()` which is unaffected